sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -1138,20 +1138,36 @@ def clear_state(self, window, tag):
     if not self.state[window]:
       self.state.pop(window, None)
 
-  def get_timers(self, clear=False, watermark=MAX_TIMESTAMP):
+  def get_timers(self, clear=False, watermark=MAX_TIMESTAMP, current_time=None):
+    """Gets expired timers and reports if there
```

Comment:
<!--thread_id:cc_155904242_t; commit:92ae785f427ba708614ab0ad110b465142150610; resolved:1-->
<!--section:context-quote-->
> **charlesccychen** wrote:
> Let's try to fit in one line:
> 
> ```python
> """Returns expired timers and existence of REAL_TIME timers."""
> ```

<!--section:body-->
Yes, it can be done for the first sentence, but I think the second is more important (explaining the addition of 'time_marker'). When I added the docstring initially, I considered writing that second sentence by the 'if' but I found it more clunky and, since the expiration is mentioned in the first sentence, referring to it in the second flowed better.